### ISSUE TITLE
Sticky navbar jolts to nav anchor on click

### ISSUE THREAD

**Issue by dylanh724** [readthedocs/sphinx_rtd_theme#1586]
## Details

* Read the Docs project URL: https://readthedocs.org/projects/acceleratxrio/
* Build URL (if applicable): https://readthedocs.org/projects/acceleratxrio/builds/25237206/
* Read the Docs username (if applicable): https://readthedocs.org/profiles/d.hunt/

## Expected Result

The left [sticky] navbar shouldn't move at all when a toctree navbar element is clicked for a smooth nav UX.

(OR, at with an option to turn this off, anchor jump top the top-level parent category; not to individual children)

## Actual Result

Navigating to new pages "jolts" the navbar, scrolling down to align with whatever child navbar toctree element was clicked. This creates a frustrating UX experience. See this low-quality GIF (apologies, the mp4 was too big in size):

![GIF](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExcnV5YTd6bnFkZ2FhM3pjeGV1cnl4ZW05cGZxZmYwMDQyOHE3dndzZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/c0P8awyC8rjLXcYNHv/giphy.gif)

## Thoughts

What I believe is happening is that click jumps to the body's selected header -- intended and good. However, if it's a *sticky* navbar, the navbar is separately jumping to its own anchor (not intended -- or if intended, it should be optional and default off since it looks glitchy)

TL;DR _Only_ the body should be anchored on click; not the navbar itself in addition.

**Comment by komaromypeter75** [comment]
Hi, any updates on this issue? Same is happening in my nav bar (rtd theme)
